API/APIManager.m

```diff
@@ -79,4 +69,48 @@ - (void)postStatusWithText:(NSString *)text completion:(void (^)(Tweet *, NSErro
     }];
 }
 
+- (void)favorite:(Tweet *)tweet completion:(void (^)(Tweet *, NSError *))completion {
+    NSString *urlString = @"1.1/favorites/create.json";
+    NSDictionary *parameters = @{@"id": tweet.idStr};
+    [self POST:urlString parameters:parameters progress:nil success:^(NSURLSessionDataTask * _Nonnull task, NSDictionary *  _Nullable tweetDictionary) {
+        Tweet *tweet = [[Tweet alloc]initWithDictionary:tweetDictionary];
```

Comment:
nit: Spacing
`[[Tweet alloc] initWithDictionary:tweetDictionary];`

Reply:
Does this initializer guarantee a tweet? It should not I'm guessing. So may want to complete with an error if there isn't a tweet.